packages/api/src/controllers/orchestrator.js

```diff
@@ -11,4 +11,28 @@ const getOrchestrators = async (req, res, next) => {
 app.get("/", getOrchestrators);
 app.get("/ext/:token", getOrchestrators);
 
-export default app;
+async function discoveryAuthWebhookHandler(req, res) {
+  const responseObj = {
+    priceInfo: {
```

Comment:
If the priceInfo response is always populated with empty strings, will the O treat this as meaning pricePerUnit = 0 and pixelsPerUnit = 0? I know that the O uses the omitempty label for the priceInfo field in the JSON response, but since we have an actual object here by default will the O accept it instead of treating it as nil?

Reply:
You're right, omitting it from the object by default would be better. fixed in a37855d